src/styles/styles.js

```diff
@@ -1191,7 +1191,6 @@ const styles = {
     avatarNormal: {
         height: variables.componentSizeNormal,
         width: variables.componentSizeNormal,
-        backgroundColor: themeColors.icon,
```

Comment:
Without removing this, the background would appear square:
![Screen Shot 2021-07-19 at 11 27 23 PM](https://user-images.githubusercontent.com/4741899/126273973-23209a95-4e40-466b-ac05-76e7cf9a5fff.png)

Reply:
Coool and it doesn't seem to affect the other icons. LGTM